src/formatDate.js

```diff
@@ -50,7 +50,70 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldSeparator = fromFormat[fromFormat.length - 1];
+  const newSeparator = toFormat[toFormat.length - 1];
+  const arrDate = date.split(oldSeparator);
+
+  const result = [];
+  let year, month, day, oldYearFormat;
+
+  for (let i = 0; i < fromFormat.length; i++) {
+    switch (fromFormat[i]) {
+      case ('YY'):
+        oldYearFormat = 2;
+        year = arrDate[i];
```

Comment:
Look, you repeat the `arrDate[i]` value not once. It's better to move it to a separate variable to improve code readability and reduces redundancy.